metrics-operator/api/v1alpha3/keptnmetric_types.go

```diff
@@ -57,12 +59,26 @@ type ProviderRef struct {
 type RangeSpec struct {
 	// Interval specifies the duration of the time interval for the data query
 	// +kubebuilder:default:="5m"
-	Interval string `json:"interval,omitempty"`
+	Interval string `json:"interval"`
```

Comment:
removing the `omitempty` tag makes those required fields - which is also the reason for the failing integration tests. Is there anything that speaks against keeping the `omitempty` tags here? If not, I would keep them

Reply:
Fixed, Thanks